.ci/install.sh

```diff
@@ -112,4 +114,6 @@ setup_shell() {
 
 check_dependencies
 download_fnm
-setup_shell
+if [ "$INSTALL_DIR" == "$HOME" ]; then 
```

Comment:
I think this is cool heuristic but I'd rather have `$SKIP_SHELL_INSTALLATION` or a `--skip-shell` flag, making it more explicit. What if you don't want it to be on your `$HOME/.fnm` but `$HOME/.my-favorite-manager`?

Reply:
That is a good point. I can add this as well. Should I also make the name of the folder (.fnm) be changable? or should I make the install dir parameter be the entire install dir and not append `.fnm` for the user?

Reply:
I think we shouldn't append `.fnm` for the user

Reply:
The user wants to configure the install dir - so we have no assumptions there! 😜